src/App.js

```diff
@@ -14,11 +16,93 @@ const goodsFromServer = [
   'Garlic',
 ];
 
-const App = () => (
-  <div className="App">
-    <h1>Selected good: -</h1>
-    {goodsFromServer.length}
-  </div>
-);
+class App extends React.Component {
+  state = {
+    selectedGoods: ['Jam'],
+  };
+
+  setTitle() {
+    const { selectedGoods } = this.state;
+
+    if (selectedGoods.length === 1) {
+      return `${selectedGoods[0]} is selected`;
+    }
+
+    if (selectedGoods.length > 1) {
+      return `${selectedGoods.join(', ')} are selected`;
```

Comment:
```suggestion
      selectedGoods.length = lengthGoods - 1;
      return `${selectedGoods.join(', ')} and ${lastGood} are selected`;
```
Если уж прямо по условию сделать, то можно так

Reply:
good point